inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/InApp.kt

```diff
@@ -137,6 +138,22 @@ internal class InApp(
         }
     }
 
+    @SuppressWarnings("TooGenericExceptionCaught")
+    override fun closeTooltip(uiElementIdentifier: String) {
+        try {
+            if (ConfigResponseRepository.instance().isConfigEnabled()) {
```

Comment:
lines 143-155 can be refactored since it is almost the same as `closeMessage` API

Reply:
refactored